Makefile

```diff
@@ -292,11 +292,11 @@ $(foreach ITEM,$(PILOT_GO_BINS_SHORT),$(eval $(call pilotbuild,$(ITEM))))
 
 # Non-static istioctls. These are typically a build artifact.
 ${ISTIO_OUT}/istioctl-linux: depend
-	STATIC=0 GOOS=linux   bin/gobuild.sh $@ istio.io/istio/pkg/version ./pilot/cmd/istioctl
+	STATIC=0 GOOS=linux   bin/gobuild.sh $@ istio.io/istio/pkg/version ./istioctl
```

Comment:
PILOT_GO_BINS and PILOT_GO_SHORT_BINS should not have istioctl, a new dependency for istioctl should be added for build target

Reply:
Yeah, it should be sufficient to remove the instances of istioctl from PILOT_GO_BINS and PILOT_GO_BINS_SHORT, then add a rule like:

<pre>istioctl ${ISTIO_OUT}/istioctl: 
	bin/gobuild.sh ${ISTIO_OUT}/istioctl istio.io/istio/pkg/version ./istioctl</pre>